docs/components/Combobox/Web.stories.tsx

```diff
@@ -436,6 +438,49 @@ const ComboboxInfiniteScroll: ComponentStory<typeof Combobox> = args => {
   );
 };
 
+const ComboboxKeepOpenOnActionClick: ComponentStory<typeof Combobox> = args => {
+  const [selected, setSelected] = useState<ComboboxOption[]>([]);
+  const [chips, setChips] = useState<string[]>([]);
+
+  const handleActionClick = (searchValue: string) => {
+    setChips([...chips, searchValue]);
+  };
+
+  return (
+    <Card header="Add more friends">
+      <Content>
+        <div style={{ display: "flex", alignItems: "center", gap: "16px" }}>
+          <Combobox
+            {...args}
+            onSelect={setSelected}
+            selected={selected}
+            label="Add Chip"
+          >
+            <Combobox.Option id="1" label="Search" />
+            <Combobox.Option id="2" label="for a" />
+            <Combobox.Option id="3" label="friend" />
+
+            <Combobox.Action
+              visible={({ searchValue }) => Boolean(searchValue)}
```

Comment:
any reason we want to have to search before exposing the action in this example?

it's for sure my fault for literally not reading the words in front of my eyes, but I was confused why there was no action when checking out this story for an action. 

normally the content in the stories is just silly names that I default to not reading it, or at least not actually processing it. it just goes in and out immediately.

Reply:
summarized our chat in the comment thread above ^